Example/Backpack/BPKAppDelegate.m

```diff
@@ -39,10 +39,15 @@ @implementation BPKAppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     self.UITestingEnabled = [NSProcessInfo.processInfo.arguments containsObject:@"UITests"];
 
-    id<BPKFontDefinitionProtocol> relativeFontDefinition = [BPKRelativeFontDefinition new];
-    UIFont *relativeTestFont = [UIFont fontWithName:relativeFontDefinition.regularFontFace size:12];
-    if (relativeTestFont != nil) {
-        [BPKFont setFontDefinition:relativeFontDefinition];
+
+    BOOL useRelative = ![NSProcessInfo.processInfo.arguments containsObject:@"DISABLE_RELATIVE"];
+
+    if (useRelative) {
+        id<BPKFontDefinitionProtocol> relativeFontDefinition = [BPKRelativeFontDefinition new];
+        UIFont *relativeTestFont = [UIFont fontWithName:relativeFontDefinition.regularFontFace size:12];
+        if (relativeTestFont != nil) {
+            [BPKFont setFontDefinition:relativeFontDefinition];
+        }
```

Comment:
Would it be cleaner if we did something like `useRelative = !env["DISABLE_RELATIVE"] && relativeAvailable`

Then underneath have 
```
if (useRelative) {
             [BPKFont setFontDefinition:relativeFontDefinition];
         }
         ```

Reply:
Like so https://github.com/Skyscanner/backpack-ios/pull/1002/commits/357f3c398d85a44a3d245732ffd6595b438ce3d3?